<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="leftMargin" >
     <number>9</number>
    </property>
    <property name="topMargin" >
     <number>9</number>
    </property>
    <property name="rightMargin" >
     <number>9</number>
    </property>
    <property name="bottomMargin" >
     <number>9</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string>Class</string>
        </property>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>3</number>
         </property>
         <property name="leftMargin" >
          <number>3</number>
         </property>
         <property name="topMargin" >
          <number>3</number>
         </property>
         <property name="rightMargin" >
          <number>3</number>
         </property>
         <property name="bottomMargin" >
          <number>3</number>
         </property>
         <item>
          <widget class="QComboBox" name="comboBox_class" >
           <property name="minimumSize" >
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="title" >
         <string>Subject</string>
        </property>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>3</number>
         </property>
         <property name="leftMargin" >
          <number>3</number>
         </property>
         <property name="topMargin" >
          <number>3</number>
         </property>
         <property name="rightMargin" >
          <number>3</number>
         </property>
         <property name="bottomMargin" >
          <number>3</number>
         </property>
         <item>
          <widget class="QComboBox" name="comboBox_subject" >
           <property name="minimumSize" >
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3" >
        <property name="title" >
         <string>Pupil</string>
        </property>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>3</number>
         </property>
         <property name="leftMargin" >
          <number>3</number>
         </property>
         <property name="topMargin" >
          <number>3</number>
         </property>
         <property name="rightMargin" >
          <number>3</number>
         </property>
         <property name="bottomMargin" >
          <number>3</number>
         </property>
         <item>
          <widget class="QComboBox" name="comboBox_pupil" >
           <property name="minimumSize" >
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_previous" >
           <property name="toolTip" >
            <string>Previous pupil</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset>icons/go-back.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_next" >
           <property name="toolTip" >
            <string>Next pupil</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset>icons/go-forward.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="GView" name="graphicsView" />
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Teacher:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_teacher" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frame" >
         <bool>false</bool>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_chars" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Click on a character to insert it in the text</string>
        </property>
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open" />
    <addaction name="action_Synchronize" />
    <addaction name="separator" />
    <addaction name="action_Quit" />
   </widget>
   <widget class="QMenu" name="menu_Edit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Copy" />
    <addaction name="action_Cut" />
    <addaction name="action_Paste" />
    <addaction name="separator" />
    <addaction name="action_Undo" />
    <addaction name="action_Redo" />
   </widget>
   <widget class="QMenu" name="menu_Report" >
    <property name="title" >
     <string>&amp;Report</string>
    </property>
    <addaction name="action_RequestCheck" />
    <addaction name="action_Print" />
    <addaction name="separator" />
    <addaction name="action_nextUnfinished" />
    <addaction name="separator" />
    <addaction name="action_autospellcheck" />
    <addaction name="action_checkSpelling" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Edit" />
   <addaction name="menu_Report" />
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="action_Copy" />
   <addaction name="action_Cut" />
   <addaction name="action_Paste" />
   <addaction name="action_Undo" />
   <addaction name="action_Redo" />
   <addaction name="separator" />
   <addaction name="action_styleN" />
   <addaction name="action_styleS" />
   <addaction name="action_styleSR" />
   <addaction name="separator" />
   <addaction name="action_indent" />
   <addaction name="action_unindent" />
   <addaction name="separator" />
   <addaction name="action_Print1" />
   <addaction name="action_Print" />
   <addaction name="separator" />
   <addaction name="action_RequestCheck" />
   <addaction name="action_nextUnfinished" />
   <addaction name="action_checkSpelling" />
   <addaction name="action_autospellcheck" />
   <addaction name="separator" />
   <addaction name="separator" />
   <addaction name="separator" />
   <addaction name="separator" />
   <addaction name="separator" />
   <addaction name="action_Quit" />
  </widget>
  <action name="action_Copy" >
   <property name="icon" >
    <iconset>icons/copy.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Copy</string>
   </property>
   <property name="toolTip" >
    <string>Copy marked text</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Cut" >
   <property name="icon" >
    <iconset>icons/cut.png</iconset>
   </property>
   <property name="text" >
    <string>Cu&amp;t</string>
   </property>
   <property name="toolTip" >
    <string>Cut marked text</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Paste" >
   <property name="icon" >
    <iconset>icons/paste.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Paste</string>
   </property>
   <property name="toolTip" >
    <string>Paste to cursor position</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Undo" >
   <property name="icon" >
    <iconset>icons/undo.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Undo</string>
   </property>
   <property name="toolTip" >
    <string>Undo last editing operation(s)</string>
   </property>
  </action>
  <action name="action_Redo" >
   <property name="icon" >
    <iconset>icons/redo.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Redo</string>
   </property>
   <property name="toolTip" >
    <string>Redo last undone editing operation</string>
   </property>
  </action>
  <action name="action_styleS" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset>icons/styleS.png</iconset>
   </property>
   <property name="text" >
    <string>Special style text</string>
   </property>
  </action>
  <action name="action_styleN" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset>icons/styleN.png</iconset>
   </property>
   <property name="text" >
    <string>Normal style text</string>
   </property>
  </action>
  <action name="action_styleSR" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset>icons/styleSR.png</iconset>
   </property>
   <property name="text" >
    <string>Special style text (right aligned)</string>
   </property>
  </action>
  <action name="action_indent" >
   <property name="icon" >
    <iconset>icons/indent.png</iconset>
   </property>
   <property name="text" >
    <string>indent</string>
   </property>
   <property name="toolTip" >
    <string>Increase line indentation</string>
   </property>
  </action>
  <action name="action_unindent" >
   <property name="icon" >
    <iconset>icons/unindent.png</iconset>
   </property>
   <property name="text" >
    <string>unindent</string>
   </property>
   <property name="toolTip" >
    <string>Decrease line indentation</string>
   </property>
  </action>
  <action name="action_Print" >
   <property name="icon" >
    <iconset>icons/print.png</iconset>
   </property>
   <property name="text" >
    <string>Print</string>
   </property>
   <property name="toolTip" >
    <string>Print all reports for this subject</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_RequestCheck" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset>icons/finished.png</iconset>
   </property>
   <property name="text" >
    <string>Report completed</string>
   </property>
   <property name="toolTip" >
    <string comment="'Down' indicates report completed" >Report completed</string>
   </property>
  </action>
  <action name="action_Open" >
   <property name="icon" >
    <iconset>icons/open.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open</string>
   </property>
   <property name="toolTip" >
    <string>Open Report Database</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Quit" >
   <property name="icon" >
    <iconset>icons/quit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_nextUnfinished" >
   <property name="icon" >
    <iconset>icons/nextUnfinished.png</iconset>
   </property>
   <property name="text" >
    <string>Next unfinished report</string>
   </property>
   <property name="toolTip" >
    <string>Go to next uncompleted report</string>
   </property>
  </action>
  <action name="action_autospellcheck" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset>icons/autospellcheck.png</iconset>
   </property>
   <property name="text" >
    <string>Auto-spellcheck</string>
   </property>
   <property name="toolTip" >
    <string>Automatic spellchecking</string>
   </property>
  </action>
  <action name="action_checkSpelling" >
   <property name="icon" >
    <iconset>icons/spellcheck.png</iconset>
   </property>
   <property name="text" >
    <string>Check spelling</string>
   </property>
   <property name="iconText" >
    <string>Check spelling</string>
   </property>
   <property name="toolTip" >
    <string>Interactive spelling checker</string>
   </property>
  </action>
  <action name="action_Print1" >
   <property name="icon" >
    <iconset>icons/print1.png</iconset>
   </property>
   <property name="text" >
    <string>Print this report</string>
   </property>
   <property name="toolTip" >
    <string>Print just this report</string>
   </property>
  </action>
  <action name="action_Synchronize" >
   <property name="icon" >
    <iconset>icons/refresh.png</iconset>
   </property>
   <property name="text" >
    <string>Synchronize</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GView</class>
   <extends>QGraphicsView</extends>
   <header>guiCanvas</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>362</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
